<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConnectionDialog</class>
 <widget class="QDialog" name="DatabaseConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Реестр 223-ФЗ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/main.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Для работы программы необходимо подключиться к базе данных.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Для этого нужно заполнить все данные.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userNameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="hostNameLabel">
       <property name="text">
        <string>Имя хоста</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostNameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userNameEdit</tabstop>
  <tabstop>hostNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatabaseConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatabaseConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
